### PR TITLE
#164069305  api documentation swagger and coreapi

### PR DESCRIPTION
**Pull request by akram256**
**What does this PR do?**
*Api documentation with swagger and core api

**Description of the tasks to be completed?**
*Update requirements with core api and rest framework swagger
*Refactor urls to add schema_views for coreapi and swagger

**How should this be manaully tested?**

* Run `http://127.0.0.1:8000/api/coreapi/`
* Run `http://127.0.0.1:8000/api/swagger/`

**What are the relevant pivot tracker stores?**
#164069305

**screenshots**
* Screenshots for api documentation using swagger
screenshot 2019-02-27 at 11 00 46

* Screenshots for api documentation using coreapi
screenshot 2019-02-27 at 11 35 19

### CODE REVIEW

**Comment by PatrickCmd** [comment]
@akram256 The url for the coreapi documentation should be pointing to the docs instead of the schema. Yes we can have both the docs and the schema but for our case, I want that url point to the api-docs of coreapi

File: authors/apps/authentication/urls.py [hunk 1]
```diff
@@ -1,11 +1,15 @@
...
-
+from rest_framework_swagger.views import get_swagger_view
+from rest_framework.documentation import include_docs_urls
 from .views import (
     LoginAPIView, RegistrationAPIView, UserRetrieveUpdateAPIView
 )
-
+coreapi_docs = include_docs_urls(title='Authors Haven')
+schema_view = get_swagger_view(title='Authors Haven')
 urlpatterns = [
```
Comment (PatrickCmd) [inline authors/apps/authentication/urls.py:7]: @akram256 Move both these documentation urls to the urls file of the project.

File: authors/apps/authentication/urls.py [hunk 2]
```diff
@@ -1,11 +1,15 @@
 from django.urls import path
-
+from rest_framework_swagger.views import get_swagger_view
+from rest_framework.documentation import include_docs_urls
```
Comment (PatrickCmd) [inline authors/apps/authentication/urls.py]: @akram256 Move these two swagger and documentation imports to the urls module of the project.

Comment (akram256) [reply authors/apps/authentication/urls.py]: I have worked on the feed back

**Review by PatrickCmd — APPROVED** [review]
LGTM

**Pull request merged**